src/domain/event/EventUtils.ts

```diff
@@ -57,9 +57,24 @@ export const isEventFree = (event: EventFieldsFragment): boolean => {
  * Get event id from url
  * For example  https://api.hel.fi/linkedcourses/v1/event/harrastushaku:13433?query -> harrastushaku:13433
  */
-export const getEventIdFromUrl = (url: string): string | null => {
-  const result = url.match(/\/event\/([^/?]*)/i);
-  return result?.[1] || null;
+export const getEventIdFromUrl = (
```

Comment:
A matter of taste but these could be in a RouterUtils.ts file or something like that? Most of the functions here are reducing results from EventFieldsFragment object. Also some tests could be added, unit tests are usually very easy to add.

Reply:
These are basically covered with integration/component tests. But maybe there are other benefits too to have tests for this small functions, faster feedback and easier to debug if big components tests are failing because of smaller functions